data/json/snippets/zombie_anatomy.json

```diff
@@ -171,6 +171,72 @@
       "You cut into its abdomen, and discover its utter lack of internal organs, the expected cavity replaced by nothing but more flesh and muscle."
     ]
   },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_grabber_general>",
+    "text": [
+      "The sheer remaining tension in each of the muscles of its arms and shoulders makes them unwind with a sonorous thwang each time you cut one of the supporting bone.",
```

Comment:
```suggestion
      "The sheer remaining tension in each of the muscles of its arms and shoulders makes them unwind with a sonorous thwang each time you cut one of the supporting bones.",
```

Reply:
Thanks for the suggestion, but I actually made a mistake with that one, will submit a new version of it shortly.